plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBExecUtils.java

```diff
@@ -853,6 +857,18 @@ public static void bindAttributes(
         }
     }
 
+    private static boolean isEnumType(DBSEntityAttribute tableColumn) {
+        if (tableColumn instanceof DBSTypedObjectEx) {
+            DBSDataType columnDataType = ((DBSTypedObjectEx) tableColumn).getDataType();
+            Object typeExt = columnDataType.geTypeExtension();
+            if (typeExt instanceof DBPNamedObject) {
+                Predicate<String> hasEnumSubstring = Pattern.compile("\\benum\\b", Pattern.CASE_INSENSITIVE).asPredicate();
```

Comment:
Kinda strange way of checking for a type, but whatever.

Please consider making this pattern a `static final` member of this class to avoid compiling it every time this method is called.

Reply:
I know. Unfortunately, there is no way to check it in a different way.